app/src/main/kotlin/com/backbase/golden_sample_app/presentation/bottom/BottomNavigationDestinations.kt

```diff
@@ -0,0 +1,51 @@
+package com.backbase.golden_sample_app.presentation.bottom
+
+import androidx.annotation.IdRes
+import androidx.navigation.NavController
+import androidx.navigation.fragment.FragmentNavigator
+import androidx.navigation.fragment.FragmentNavigatorDestinationBuilder
+import com.backbase.android.design.header.TabHeaderFragment
+import com.backbase.android.design.header.TabListConfiguration
+import com.backbase.android.design.header.TopBarConfiguration
+import com.backbase.golden_sample_app.R
+import com.backbase.golden_sample_app.presentation.header.TabListConfigurationProvider
+import com.backbase.golden_sample_app.presentation.header.TopBarConfigurationProvider
+
+fun NavController.addBottomBarNavigationDestinations(
+    tabListConfig: TabListConfigurationProvider,
+    topBarConfig: TopBarConfigurationProvider
+) = graph.addDestinations(
+    createDestination(
+        id = R.id.dashboard,
+        topBar = topBarConfig.config(),
+        tabList = tabListConfig.dashboardTabList()
+    ),
+    createDestination(
+        id = R.id.move_money,
+        topBar = topBarConfig.config(),
+        tabList = tabListConfig.moveMoneyTabList()
+    )
+)
+
+@Suppress("DEPRECATION")
```

Comment:
Suppression should be at the exact line you want to suppress to not accidentally suppress other deprecations. Also reason for suppression should be explained in the project

Reply:
For me, adopting this deprecated api while there is a non deprecated way of using (xml) is also a hard sell. In general in Android they are not quick to remove deprecated api's, but they do state it is out of support and I don't think there is a list of when certain deprecations will be removed.